examples/angular/sample-app/src/index.html

```diff
@@ -7,25 +7,22 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <script>
+    window.RudderSnippetVersion = '3.0.1';
```

Comment:
@MoumitaM, can you revert the "chore" inclusion change we made to the release scripts and add a comment in the "publish new release" workflow to keep using the GitHub runners?

Reply:
addressed